themes-default/slim/views/config_postProcessing.mako

```diff
@@ -124,14 +124,22 @@ window.app = new Vue({
             // Disable the save button until we're done.
             this.saving = true;
 
-            let config = {
+            const config = {
                 postProcessing: this.postProcessing,
                 metadata: {
                     metadataProviders: this.metadataProviders
                 }
             };
 
-            $store.dispatch('setConfig', {section: 'main', config: config}).then(() => {
+            // Clone the config into a new object
+            let configCopy = Object.assign({}, config);
+
+            // Use destructuring to remove the unwanted keys.
+            const { multiEpStrings, reflinkAvailable, ...rest} = configCopy.postProcessing
```

Comment:
Won't this cause any issue with `xo` or `ava`?

Reply:
Ooh i dont know. Its valid destructuring. I checked. And our test are okay for now.